### PR TITLE
<Layout /> - changed default gap for reduced spacing feature

### PR DESCRIPTION
**Pull request by egidijusja**
### 🔦 Summary

...

### ✅ Checklist

- [ ] 👨‍💻 API change is approved by the UI Infra Developers 
- [ ] 👨‍🎨 UX change is approved by the Design System UX 
- 📚 Change is documented
  - [ ] Story
  - [ ] API description
  - [ ] Cheatsheet
  - [ ] Other (explain)
- 🔬 Change is tested
  - [ ] Component
  - [ ] Visual test

### CODE REVIEW

**Comment by WixBuildServer** [comment]
### 🤖 Hi, I'm #Janet!

  A deploy preview of your PR was successfully released 🖼.

  Project Name: **wix-ui-tpa**

  You can find it under one of the following URLs:
  * https://bo.wix.com/pages/wix-ui-tpa.pr_6935/

**Comment by WixBuildServer** [comment]
### 🤖 Hi, I'm #Janet!

  A deploy preview of your PR was successfully released 🖼.

  Project Name: **wix-style-react**

  You can find it under one of the following URLs:
  * https://preview.wix-style-react.com/pr_6935

File: packages/wix-style-react/src/Layout/Layout/index.js [hunk 1]
```diff
@@ -11,18 +15,29 @@ const Layout = ({
...
-    className={classes.root}
-    children={children}
-  />
+  <WixStyleReactContext.Consumer>
+    {({ reducedSpacingAndImprovedLayout }) => (
+      <div
+        data-hook={dataHook}
+        style={{
+          gap:
+            gap === DEFAULT_GAP
```
Comment (mykas) [inline packages/wix-style-react/src/Layout/Layout/index.js:24]: I believe this should be enough, no?
```
gap: gap? gap: reducedSpacingAndImprovedLayout? REDUCED_SPACING_GAP: DEFAULT_GAP;
```
Then we can remove the default prop for gap.

Comment (egidijusja) [reply packages/wix-style-react/src/Layout/Layout/index.js:24]: I thought about this as well, but with this solution we will not show default value for gap on our storybook API section. Is it okay?

Comment (mykas) [reply packages/wix-style-react/src/Layout/Layout/index.js:24]: Dammit it, storybook! :// ok..

Comment (mykas) [reply packages/wix-style-react/src/Layout/Layout/index.js:24]: Maybe there another way to put it in some variable and make it more readable?

**Review by mykas — APPROVED** [review]
Nicely done! @milkyfruit just FYI - this is I believe the last component that was missing experiment `reducedSpacingAndImprovedLayout`

**Review by milkyfruit — APPROVED** [review]

**Comment by WixBuildServer** [comment]
### 🤖 Hi, I'm #Janet!

A deploy preview of your PR was successfully released 🖼.

Project Name: **wix-ui-tpa**

You can find it under one of the following URLs:
* https://bo.wix.com/pages/wix-ui-tpa.pr_6935/

**Comment by WixBuildServer** [comment]
### 🤖 Hi, I'm #Janet!

A deploy preview of your PR was successfully released 🖼.

Project Name: **wix-style-react**

You can find it under one of the following URLs:
* https://preview.wix-style-react.com/pr_6935

**Pull request merged**